src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1635,12 +1639,16 @@ def client_side_proxy_wrapper(cmd,
         requestUri = f'{consts.CSP_Storage_Url}/{consts.RELEASE_DATE_WINDOWS}/arcProxy{operating_system}{consts.CLIENT_PROXY_VERSION}.exe'
         older_version_string = f'.clientproxy\\arcProxy{operating_system}*.exe'
         creds_string = r'.azure\accessTokens.json'
+        msal_token_cache_user = r'.azure\msal_token_cache.bin'
+        msal_token_cache_spn = r'.azure\service_principal_entries.bin'
 
     elif(operating_system == 'Linux' or operating_system == 'Darwin'):
         install_location_string = f'.clientproxy/arcProxy{operating_system}{consts.CLIENT_PROXY_VERSION}'
         requestUri = f'{consts.CSP_Storage_Url}/{consts.RELEASE_DATE_LINUX}/arcProxy{operating_system}{consts.CLIENT_PROXY_VERSION}'
         older_version_string = f'.clientproxy/arcProxy{operating_system}*'
         creds_string = r'.azure/accessTokens.json'
+        msal_token_cache_user = r'.azure/msal_token_cache.bin'
+        msal_token_cache_spn = r'.azure/service_principal_entries.bin'
```

Comment:
DON'T DO THIS. THIS IS NOT SUPPORTED. See https://github.com/Azure/azure-cli/pull/19853#issuecomment-984440373